lib/components/user/back-to-trip-planner.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+
+const Container = styled.div``
+
+const BackToTripPlanner = () => (
+  <Container>
+    <Link to='/'>← Back to trip planner</Link>
```

Comment:
This arrow is very difficult to read. We need a taller arrow, especially for the mobile view.

Reply:
The link is losing all query params. If a user had an active search prior to clicking this button, the search will be lost.